test/graphql/types/Organization/organizationResolvers.test.ts

```
@@ -0,0 +1,180 @@
+	beforeEach(() => {
+		// Reset all mocks before each test
+		vi.clearAllMocks();
+		(mockContext.drizzleClient.query.organizationsTable.findMany as jest.Mock).mockReset();
+	});
+
+	const executeQuery = async (query: string, variables = {}) => {
+		return OrganizationUpdaterResolver({
+			source: {},
+			args: variables,
+			context: mockContext,
+			info: {} as any, // Replace with appropriate GraphQLResolveInfo if available
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)